### PR TITLE
Express mongoose boilerplate

### PR DESCRIPTION
**Pull request by hyperkarmic**
**Engineer Story**

As a software engineer, I'd like to install dependencies - so I can use  Mongoose and CORS functionalities on my project.

I need to set up a 'server file' so I can run a server(express) to set up routes so my application can send and receive data, including web-pages.

**Acceptance Criteria (ACs):**

-Package JSON file to be created.

-CORS and Mongoose to be set up by NPM, via node.

-a server file to be set up, including the import of CORS/Mongoose/Express Dependencies.  An instance of Express is to be set up.  Relevant configuration information is to be imported from the .config file.  Middleware is to be implemented.  A 'mongoose connection' is to  be set up.  A test 'get' route is to be set up, as well as a 'app.listen' method that will confirm that a connection has been established.

-I have installed dependencies using NPM on node
-I have created a server.js file, using a boiler-plate template and adapting it to my needs
-I have created a config.js file - and changed specific details to make them relevant to my project.
-Have tested the server, to make sure that it works, by using the 'test route'
-Have added a line - as suggested
-I have deleted the boiler plate file as suggested

### CODE REVIEW

**Comment by surajverma2587** [comment]
Can you delete the `package-lock.json` and add-commit-push your changes.

**Comment by surajverma2587** [comment]
Can you please add some description to your PR. Follow the template [here](https://github.com/surajverma2587/just-ate-server/pull/1)

File: src/index.js [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+
+app.use(cors());
+
+mongoose.connect(DB_URI, MONGOOSE_OPTIONS);
+
+app.get('/ping', (_, res) => {
+  res.send('pong');
+});
```
Comment (surajverma2587) [inline src/index.js:18]: Add a new line here. This is just a test comment but please do it as well :)

Comment (hyperkarmic) [reply src/index.js:18]: e48819ef64d3ed2a5258e5bbaeede25a33c5c623 I have addressed your feedback here.

**Review by surajverma2587 — APPROVED** [review]

**Pull request merged**